src/bootstrap/src/lib.rs

```diff
@@ -410,6 +410,25 @@ forward! {
     download_rustc() -> bool,
 }
 
+/// A mostly temporary helper struct before we can migrate everything in bootstrap to use
+/// the concept of a build compiler.
+struct HostAndStage {
+    host: TargetSelection,
+    stage: u32,
+}
```

Comment:
Remark: ah, I was curious what the tuple was, this makes sense on a temporary basis.